src/gwt/src/org/rstudio/studio/client/workbench/views/environment/dataimport/DataImportOptionsUiSav.java

```diff
@@ -119,6 +120,16 @@ public void setImportLocation(String importLocation)
       }
    }
    
+   @Override
+   public HelpLink getHelpLink()
+   {
+      return new HelpLink(
+         "Reading statistical data using haven",
```

Comment:
I'm sort of iffy about calling this 'statistical data' (since it somewhat implies the data you can't read with haven is not statistical).

What about something like:

```
Reading data using haven
Reading foreign datasets using haven
```

or otherwise?